cli/dcoscli/data/help/package.txt

```diff
@@ -56,6 +57,8 @@ Options:
         Command line only.
     --config
         Print the configurable properties of the `marathon.json` file.
+    --dry-run, -n
```

Comment:
Yea `dcos-docker` is the way to go. I'll get someone to take a look at those prs.